backend/src/main/java/de/silvia/backend/repositories/IArtikelListRepo.java

```diff
@@ -0,0 +1,18 @@
+package de.silvia.backend.repositories;
+
+import de.silvia.backend.models.ArtikelList;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface IArtikelListRepo extends MongoRepository<ArtikelList, String> {
+
+    Optional<ArtikelList> findArtikelListByListName(String listName);
+
+    void insert(Optional<ArtikelList> optionalArtikelList);
```

Comment:
Ich würde nicht versuchen ein Optional<ArtikelList> in die DB zu speichern. Ich würde vorher prüfen ob das Optional überhaupt eine Liste enhält und dann dieses speichern.

Reply:
Ich hab gerade gemerkt, dass ich die Funktion gar nicht benutze, hab sie entfernt. Den Hinweis mit den Optional behalte ich im Hinterkopf, Danke.